tests/Unit/Http/Action/Security/LogInActionTest.php

```diff
@@ -109,10 +110,29 @@ public function rendersLogInFormIfAuthenticationFailed()
         $this->assertSame($content, $response->getContent());
     }
 
+    /**
+     * Data provider for redirectsToDashboardIfAuthenticationSucceeded
+     *
+     * @return array
+     */
```

Comment:
Can we remove the entire DocBlock?

When we rename the test method, it becomes outdated, and it should be clear from the test method itself that it is used there.

Reply:
I was following what appeared to be a convention elsewhere. Easy enough though.